src/components/input/InputElement.js

```diff
@@ -0,0 +1,29 @@
+import React, { useState, useEffect } from 'react';
+
+
+const InputElement = ({ label, type, id, placeholder, description }) => {
+    const [inputValue, setInputValue] = useState('');
+    console.log(inputValue);
+
+    return (
+        <>
+            {
+                label &&
+                <label htmlFor={id}>{label} <br /></label>
+            }
+                <input 
+                    className="form-control"
+                    value={inputValue} 
+                    type={type} 
+                    id={id} 
+                    placeholder={placeholder}
+                    onChange={(ev) => setInputValue(ev.target.value)} />
```

Comment:
remember that the function executed should be received via props, because the component that implement this component is on charge to handle the states.